src/actions.py

```diff
@@ -123,6 +131,16 @@ def save_project_action(_action, _target, self):
     ui.save_project_dialog(self)
 
 
+def smoothen_settings_action(_action, _target, self):
+    def _on_reset(_button, self):
+        params = self.get_settings("actions").get_child("smoothen")
+        for key in params.list_keys():
+            params.reset(key)
```

Comment:
SInce this shares functionality with the import settings, perhaps there could be a way to move it to utilities? or even to vala, that way we can make direct use of builder callbacks